parquet-hadoop/pom.xml

```diff
@@ -95,6 +95,11 @@
       <artifactId>jackson-databind</artifactId>
       <version>${jackson-databind.version}</version>
     </dependency>
+    <dependency>
+      <groupId>${jackson.groupId}</groupId>
```

Comment:
Can we avoid adding dependency?

Reply:
The jackson-annotations dependency is used in parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ParquetMetadata.java . Do not serialize `InputFile inputFile` to json and keep the the same behavior as before. I'm sorry I'm not familiar with jackson library and not sure is there any other way to do this.

Reply:
I happened to find that we have a [parquet-jackson](https://github.com/apache/parquet-mr/blob/master/parquet-jackson/pom.xml#L35-L46) module which shades jackson-core and jackson-databind. But in the [parquet-hadoop](https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/pom.xml#L82-L97) (and other modules) it also explicitly depends on parquet-jackson and jackson-xxx at the same time. I'm not familiar with this history, do you know why? @gszadovszky @Fokko @shangxinli

Reply:
@wgtmac, the [README](https://github.com/apache/parquet-mr/blob/master/parquet-jackson/README.md) of parquet-jackson describes how it works. This is only for doing the shading once (and having one shaded jar) instead of in all the modules which requires jackson.

Reply:
Thanks! Sorry for missing that.